### PR TITLE
fix: fixed storage class and count processing

### PR DESCRIPTION
**Pull request by Isteb4k**
## Description
Fixed storage class and count processing

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

### CODE REVIEW

File: tests/performance/Taskfile.yaml [hunk 1]
```diff
@@ -29,14 +31,33 @@ tasks:
...
+          NAMESPACE: "{{ .NAMESPACE }}"
+          STORAGE_CLASS: "{{ .STORAGE_CLASS }}"
+          NAME_PREFIX: "{{ .NAME_PREFIX }}"
+          RESOURCES_PREFIX: "{{ .RESOURCES_PREFIX }}"
+      - task: apply:vms
+        vars:
+          COUNT: "{{ .COUNT }}"
+          NAMESPACE: "{{ .NAMESPACE }}"
+          NAME_PREFIX: "{{ .NAME_PREFIX }}"
+          RESOURCES_PREFIX: "{{ .RESOURCES_PREFIX }}"
```
Comment (fl64) [inline tests/performance/Taskfile.yaml:46]: why duplicate variables if they are already globally declared?

File: tests/performance/Taskfile.yaml [hunk 2]
```diff
@@ -29,14 +31,33 @@ tasks:
...
 
   destroy:all:
     desc: "Destroy disks and virtual machines in two steps (in two different releases)."
     cmds:
-      - task destroy:vms
-      - task destroy:disks
+      - task: destroy:vms
+        vars:
```
Comment (fl64) [inline tests/performance/Taskfile.yaml:52]: same question here

File: tests/performance/Taskfile.yaml (hunk 1 above)
Comment (Isteb4k) [reply tests/performance/Taskfile.yaml:46]: When passing variables to the task (COUNT=5 task apply:all), in subtasks, the values of global variables remain unchanged; it's necessary to explicitly pass new values ​​for the variables.

File: tests/performance/Taskfile.yaml (hunk 2 above)
Comment (Isteb4k) [reply tests/performance/Taskfile.yaml:52]: same answer here

File: tests/performance/Taskfile.yaml (hunk 1 above)
Comment (fl64) [reply tests/performance/Taskfile.yaml:46]: :okay:

**Review by universal-itengineer — APPROVED** [review]

**Pull request merged**